src/core.js

```diff
@@ -689,15 +687,14 @@ define([
         return $modal;
     };
 
-    fn._setup_fancybox = function () {
-        $.fancybox.init();
-        this.$f.find("a.inline").fancybox({
-            hideOnOverlayClick: false,
-            hideOnContentClick: false,
-            enableEscapeButton: false,
-            showCloseButton : true,
-            onClosed: function() {}
-        });
+    fn._showPageSpinner = function() {
+        var spinner = $("<div><div><div></div></div></div>");
+        spinner.addClass("fd-form-saving");
+        $('body').append(spinner);
+    };
+
+    fn._hidePageSpinner = function() {
```

Comment:
I do prefer to keep things private until they're needed publicly. But you're right, real privacy is better than fake privacy. Updated.